### PR TITLE
Create SoC status for SM6125

### PR DESCRIPTION
**Pull request by MarijnS95**
Something I created a very long time ago, need to see how accurate the numbers are and what more fields can be filled in. Submitting now _as draft_ so that I don't forget **again**.

### CODE REVIEW

File: _soc/sm8150.md [hunk 1]
```diff
@@ -50,5 +50,3 @@ status-watchdog: 5.6
 pmic: pm8150, pm8150b, pm8150l, pm8009
 ---
 Snapdragon 855 (Hana)
-
-
```
Comment (lumag) [inline _soc/sm8150.md:54]: irrelevant changes ;-)

File: _soc/sm6125.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+status-camera:
+status-camera-csi:
+status-camera-datapath:
+status-camera-eva:
+status-camera-i2c:
+status-camera-sfe:
+status-camera-vfe:
+status-camera-vfelight:
+status-clock-gcc: 5.14
+status-clock-rpmhcc: 5.15 # TODO rpmcc without H
```
Comment (lumag) [inline _soc/sm6125.md:24]: It it RPM(h)CC in the web page.

File: _soc/sm6125.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+status-display-hdmi: N/A
+status-display-hdmiaudio: N/A
+status-display-hdmicec: N/A
+status-display-gpu: WIP
+status-display-lvds: N/A
+status-dma-bam:
+status-dma-gpi: 6.3 # When it was added to DT
+status-gnss:
+status-hwspinlocks:
+status-i2c: 6.3
```
Comment (MarijnS95) [inline _soc/sm6125.md:54]: Hmm I don't think I updated this file all the way to the bottom. i2c and pinctrl were definitely there before 6.3.

File: _soc/sm6125.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+status-clock-rpmhcc: 5.15 # TODO rpmcc without H
+status-clock-tcsrcc:
+status-connectivity-bluetooth:
+status-connectivity-ethernet:
+status-connectivity-ipa:
+status-connectivity-wlan:
+status-cpu-bwmon:
+status-cpu-cachetop:
+status-cpu-cpufreq: WIP
+status-cpu-cpuidle:
```
Comment (lumag) [inline _soc/sm6125.md:33]: I think it supports PSCI, doesn't it? In such case you should have PSCI idle states.

**Review by lumag — CHANGES_REQUESTED** [review]
Please review status properties

**Pull request closed without merging**